doc/partitioned-db.md

```diff
@@ -0,0 +1,206 @@
+# Partitioned Database
+
+When creating a Cloudant database, you can configure it as non-partitioned or partitioned.
+A partitioned database offers significant performance and cost advantages but requires you to specify a logical partitioning of your data.
+
+Reference:
+
+- The driver module `nodejs-cloudant`: https://github.com/cloudant/nodejs-cloudant#partitioned-databases
+- The Cloudant official document: https://cloud.ibm.com/docs/services/Cloudant/guides?topic=cloudant-database-partitioning
+
+
+Let's start with a user scenario to understand what's been added in the new feature.
```

Comment:
It might be better to have a short description of what you need to do in order to make use of the partitioned database before giving an example.